src/app/admin/auditLogTable/auditLogModal/auditLogModalRows/Client.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+
+import React from "react";
+import supabase from "@/supabaseClient";
+import { logErrorReturnLogId } from "@/logger/logger";
+import LinkButton from "@/components/Buttons/LinkButton";
+import { AuditLogModalRowResponse } from "../types";
+import AuditLogModalRow from "../AuditLogModalRow";
+
+interface ClientLinkDetails {
+    clientId: string;
+    clientName: string;
+}
+
+type ClientLinkDetailsErrorType = "failedClientFetch";
+
+interface ClientLinkDetailsError {
+    type: ClientLinkDetailsErrorType;
+    logId: string;
+}
+
+const fetchClientLinkDetails = async (
+    clientId: string
+): Promise<AuditLogModalRowResponse<ClientLinkDetails, ClientLinkDetailsError>> => {
+    const { data: data, error } = await supabase
+        .from("clients")
+        .select("primary_key, full_name")
+        .eq("primary_key", clientId)
+        .limit(1)
```

Comment:
not fussed, but i don't think we need this `limit(1)` since we're matching by primary key (same in other places)